VERSION.m4

```diff
@@ -137,10 +137,20 @@ ifelse(IPA_VERSION_IS_GIT_SNAPSHOT, yes,
 IPA_GIT_VERSION),
 NEWLINE)) dnl IPA_VERSION end
 
+########################################################
+# Version of MIT krb5 used to build IPA
+########################################################
+define(IPA_KRB5_VERSION, translit(esyscmd(krb5-config --version | awk '{ print $NF }'), NEWLINE))
+define(IPA_KRB5_VERSION_MAJOR, translit(esyscmd(echo IPA_KRB5_VERSION | cut -d. -f1), NEWLINE))
```

Comment:
Using the `regexp()` function here would have been cleaner, but it results in an empty string. This is properly because `[]` characters are not properly escaped when M4 code is included from autoconf.